pkg/ccl/backupccl/backup_test.go

```diff
@@ -3735,6 +3731,55 @@ func TestRestoreAsOfSystemTime(t *testing.T) {
 	})
 }
 
+func TestEmptyBackupsInChain(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	const numAccounts = 10
+	_, sqlDB, _, cleanupFn := backupRestoreTestSetupWithParams(t, singleNode, numAccounts,
+		InitManualReplication, base.TestClusterArgs{
+			ServerArgs: base.TestServerArgs{
+				DefaultTestTenant: base.TestTenantAlwaysEnabled,
+			},
+		})
+	defer cleanupFn()
+	var tsBefore string
+	backupPath := "nodelocal://1/create_and_drop"
+
+	t.Run("empty-full-non-empty-inc", func(t *testing.T) {
+		sqlDB.Exec(t, "CREATE DATABASE create_and_drop")
+		sqlDB.Exec(t, "CREATE DATABASE create_and_drop_restore")
+		sqlDB.Exec(t, `BACKUP DATABASE create_and_drop INTO $1 WITH revision_history`, backupPath)
+		sqlDB.Exec(t, "CREATE TABLE create_and_drop.a (k int, v string)")
+		sqlDB.Exec(t, "INSERT INTO create_and_drop.a VALUES (1, 'foo')")
+		sqlDB.QueryRow(t, "SELECT cluster_logical_timestamp()").Scan(&tsBefore)
+		sqlDB.Exec(t, "DROP TABLE create_and_drop.a")
```

Comment:
curious: why did you need to drop the table?

Reply:
don't necessarily have to, just copy pasted this test from another test where I initially noticed this behaviour